data/vocab/bflc.ttl

```diff
@@ -0,0 +1,60 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix lite: <http://bibfra.me/vocab/lite/> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix schema: <http://schema.org/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix bf: <http://id.loc.gov/ontologies/bibframe/> .
+@prefix bflc: <http://bibfra.me/vocab/bflc/> .
+@prefix bflcext: <http://id.loc.gov/ontologies/bflc/> .
```

Comment:
as "bflc" is already used, I named this "bflcext"

Reply:
What does bflc stand for?

Reply:
bflc = BIBFRAME LC Extension 
LC = Library of Congress